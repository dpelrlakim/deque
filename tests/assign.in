newInit a
[]= a 1 37
pop_front a
[] a 0
[]= a 0 29
[] a 0
