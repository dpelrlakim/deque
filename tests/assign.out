a: 2 3 5 7 11 13 17 19
a: 2 37 5 7 11 13 17 19
a: 37 5 7 11 13 17 19
37
a: 37 5 7 11 13 17 19
a: 29 5 7 11 13 17 19
29
a: 29 5 7 11 13 17 19
